### PR TITLE
handle the returned error

### PR DESCRIPTION
**Pull request by wongearl**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
handle the returned error 
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -278,7 +278,10 @@ func (b *glusterfsMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs) e
 	}
 
 	// Cleanup upon failure.
-	mount.CleanupMountPoint(dir, b.mounter, false)
+	err = mount.CleanupMountPoint(dir, b.mounter, false)
```
[the remark this answers is not in the document]
Comment (wongearl) [reply pkg/volume/glusterfs/glusterfs.go:281]: ok

**Comment by Abirdcfly** [comment]
https://github.com/kubernetes/kubernetes/blob/89aaf7c02dd33415ac994cecfddca7c02ed75834/pkg/volume/cephfs/cephfs.go#L221-L262

I'm not very familiar with the storage part of the code, but I see a similar structure, note **line 258** here, again after failure, the output of `CleanupMountPoint` is ignored and the previous error is returned.

So I suspect that this is intentional and not a bug, but this judgment needs to be confirmed by the storage people.

**Comment by humblec** [comment]
Look like the author closed it.

**Pull request closed without merging**